<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KadasPluginManagerBase</class>
 <widget class="QFrame" name="KadasPluginManagerBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="5">
    <widget class="QToolButton" name="btnClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="close">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="7" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelFrom">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="labelBasename">
     <property name="text">
      <string>Basename</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" rowspan="2">
    <widget class="QLineEdit" name="lineEditBasename">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>isochrones</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="2">
    <widget class="QLabel" name="labelReachabiliyMode">
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2">
    <widget class="QComboBox" name="comboBoxReachibiliyMode"/>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="labelIntervals">
     <property name="text">
      <string>Intervals</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" rowspan="2">
    <widget class="QLineEdit" name="lineEditIntervals">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>10;20;30</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" rowspan="2">
    <widget class="QLabel" name="labelWith">
     <property name="text">
      <string>With</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" rowspan="2">
    <widget class="QComboBox" name="comboBoxVehicles"/>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="labelRemovePrevious">
     <property name="text">
      <string>Remove previous isoline</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QCheckBox" name="checkBoxRemovePrevious">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btnCalculate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
